src/main/java/org/jenkinsci/plugins/mesos/MesosCloud.java

```diff
@@ -470,7 +470,7 @@ public boolean configure(StaplerRequest request, JSONObject object)
 
               containerInfo = new MesosSlaveInfo.ContainerInfo(
                   containerInfoJson.getString("type"),
-                  containerInfoJson.getString("dockerImage"), volumes, parameters, networking, portMappings);
+                  containerInfoJson.getString("dockerImage"), volumes, parameters, networking, portMappings, containerInfoJson.getBoolean("dockerPrivilegedMode"));
```

Comment:
can you make the privileged mode as argument 3 to be next to "type" and "dockerImage"?

also, not yours, but mind putting each argument on its own line?